element_deeplabcut/model.py

```diff
@@ -610,22 +444,26 @@ class BodyPartPosition(dj.Part):
         """
 
     def make(self, key):
-        """.populate() method will launch training for each PoseEstimationTask"""
+        """.populate() method will launch training for each PoseEstimationTask
+        """
         from .readers import dlc_reader
 
         # ID model and directories
-        dlc_model = (Model & key).fetch1()
+        dlc_model = (_linking_module.model.Model & key).fetch1()

```

Comment:
```suggestion
        dlc_model = (Model & key).fetch1()

```
Why the need to access `Model` via `_linking_module` and not just directly?